app/Http/Controllers/Api/ApiLinksEditController.php

```diff
@@ -0,0 +1,115 @@
+<?php
+
+namespace App\Http\Controllers\Api;
+
+use Illuminate\Http\Request;
+
+use App\Models\Link;
+
+class ApiLinksEditController extends ApiController
+{
+	public static function listLinks(Request $request) {
+		/**
+		 * List all (shortened) links created by the authenticated user
+		 *
+		 * @param Request $request
+		 * @return mixed
+		 */
+		//
+		// Initialize incoming parameters
+		//
+		$response_type = $request->input('response_type');
+
+		// Initialize user
+		$user = self::getApiUserInfo($request);
+
+		//
+		// Fetch valid links belonging to the same user
+		//
+		$links = Link::where('creator', $user->username)->get();
```

Comment:
Could we paginate this? (e.g, if a user has 20000 links, we wouldn't want to crash the server)